src/tests/api/controllers/order_controller_integration_test.py

```diff
@@ -110,3 +111,17 @@ def test_update_order_from_repository_using_controller(self):
 
         self.assertEqual(len(orders), 2)
         self.assertEqual(updated_order.order_details, order_to_update.order_details)
+
+    def test_get_orders_to_process_successfully(self):
+
+        order_1 = build_order()
+        order_2 = build_order()
+        order_3 = build_order(order_status=OrderStatus.COMPLETED)
+
+        self.order_controller.add(order_1)
+        self.order_controller.add(order_2)
+        self.order_controller.add(order_3)
+
+        orders_to_process = self.order_controller.get_orders_to_process()
+
+        self.assertEqual(orders_to_process, [order_1, order_2])
```

Comment:
Could you assert the call in the repository layer?

Reply:
done.